release-tasks.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "*** Running any new migrations ***"
+rake db:migrate
```

Comment:
I don't remember which version it was introduced, but `rails` [is now the default command](https://guides.rubyonrails.org/command_line.html) instead of using `rake`. They do the same thing in this case, but I've been slowly adapting to the new way. I'd probably also add a `bundle exec` to prevent any environment weirdness:

```suggestion
bundle exec rails db:migrate
```

Reply:
Hmm, actually, I decided to go with rake over rails because I wanted to avoid any issues with this being a bash executable.  But if rails will work fine then I'll switch it.

Reply:
@rlgreen91 I can't think of a scenario where this would work with `rake` but `rails` would not be available. The `db:migrate` task depends on the entire Rails environment being loaded, so it won't work unless Rails is present.

That being said, I don't think there's any practical difference, so no biggie if you want to leave it as `rake`.

Reply:
I'd say let's go with `rails` if it works now.